app/models/reminder_activity.rb

```diff
@@ -1,2 +1,6 @@
 class ReminderActivity < Activity
+  def self.already_reminded_appointment_ids
+    where(created_at: (Time.zone.now - 48.hours)..Time.zone.now)
```

Comment:
Do you reckon `where(created_at: 48.hours.ago.in_time_zone..Time.zone.now)` is any more concise?